### PR TITLE
scripts/genproto: run "protodoc" by default

### PR DESCRIPTION
**Pull request by gyuho**
Do not skip "protodoc" by default.

@hexfusion @yudai

### CODE REVIEW

File: scripts/genproto.sh [hunk 1]
```diff
@@ -114,10 +114,10 @@ schwag -input=Documentation/dev-guide/apispec/swagger/rpc.swagger.json
 # install protodoc
 # go get -v -u github.com/coreos/protodoc
 #
-# by default, do not run this option.
-# only run when './scripts/genproto.sh -g'
+# run './scripts/genproto.sh skip-protodoc'
+# to skip protodoc generation
 #
-if [ "$1" = "-g" ]; then
+if [ "$1" != "skip-protodoc" ]; then
```
Comment (hexfusion) [inline scripts/genproto.sh]: @gyuho I agree with this I have never ran the script without the intention of protodoc gen.

**Comment by hexfusion** [comment]
lgtm!

File: scripts/genproto.sh [hunk 2]
```diff
@@ -114,10 +114,10 @@ schwag -input=Documentation/dev-guide/apispec/swagger/rpc.swagger.json
 # install protodoc
 # go get -v -u github.com/coreos/protodoc
 #
-# by default, do not run this option.
-# only run when './scripts/genproto.sh -g'
+# run './scripts/genproto.sh skip-protodoc'
```
Comment (fanminshi) [inline scripts/genproto.sh]: will `--skip-protodoc` matches the convention of command flag?

Comment (gyuho) [reply scripts/genproto.sh]: @fanminshi Yeah just changed it to `--skip-protodoc`.

**Pull request merged**